ext/FluxReactantExt.jl

```diff
@@ -186,13 +177,15 @@ function Flux.gradient(f::Function, m::Fluxactor, xs::Const...)
     elseif input == m.grad_input
         # m.grad_compiled(Reverse, f, Active, dup, xrs...)
         m.grad_compiled(Reverse, Const(_fun!), seed, Const(f), dup, xrs...)
+        m.grad_count += 1
     else
-        @info "compiling gradient($f, ::Fluxactor)..."
+        @info "compiling gradient($f, ::Reactor, ::Const...)"
         # fun = @compile Enzyme.autodiff(Reverse, f, Active, dup, xrs...)  # this gives ERROR: "Unhandled type Type" above
         fun = @compile Enzyme.autodiff(Reverse, Const(_fun!), seed, Const(f), dup, xrs...)  # this gives ERROR: type TypeVar has no field data
```

Comment:
Yeah this is a bit of an aside, but presently you can’t directly compile autodiff, autodiff needs to be inside of another function.

Reply:
Ok. This I changed in [0665a55](https://github.com/FluxML/Fluxperimental.jl/pull/28/commits/0665a55d83085ce4da61eef320dd3ab66c95d333) and get different errors :) 

(If `@compile` requires exactly one function call with Symbol arguments, ideally the macro would check that & reject more complicated syntax immediately?)